README.md

```diff
@@ -1,15 +1,27 @@
 # bitmovin-player-tizen-demo
-This repository contains a sample app that used the Bitmovin HTML5 Player in a Tizen App. 
+
+This repository contains a sample app that used the Bitmovin HTML5 Player in a Tizen App. Project structure:
+
+| File/Folder | Description                                                                           |
+| ----------- | ------------------------------------------------------------------------------------- |
+| context.xml | Contains configuration options, application priviliges and entry points               |
+| index.html  | Html laying out the structure of the demo and definition of the used player resources |
+| js/main.js  | main javascript file our demo applicaiton will use                                    |
+| images/     | place for the application logo                                                        |
+| css/        | stylesheets used for making the demo application pretty                               |
```

Comment:
either start all entries in "Description" with upper or lower case but don't mix :)

Reply:
Done :heavy_check_mark: [`f329268` (#12)](https://github.com/bitmovin/bitmovin-player-tizen-demo/pull/12/commits/f3292687af78627a0b69c275140d48b1f732a299)